region/crateria/central.json

```diff
@@ -5575,13 +5621,180 @@
+                          "fromNode": 1,
+                          "usedTiles": 12
+                        }},
+                        {"spikeHits": 1}
+                      ]},
+                      {"and": [
+                        "can3HighMidAirMorph",
+                        {"canComeInCharged": {
+                          "framesRemaining": 180,
+                          "fromNode": 1,
+                          "unusableTiles": 13
+                        }},
+                        {"spikeHits": 1}
+                      ]},
+                      {"and": [
+                        {"canShineCharge": {
+                          "usedTiles": 21,
+                          "openEnd": 2
+                        }},
+                        {"adjacentRunway": {
+                          "fromNode": 1,
+                          "usedTiles": 4
+                        }},
+                        {"spikeHits": 2}
+                      ]}
+                    ]},
+                    "canSpeedball"
+                  ],
+                  "note": [
+                    "Bounce into the spikes and use a SpeedKeep to run on spikes to setup for a speedball towards the item.",
+                    "Bouncing on the platform near the door can save a spike hit, if the running space is available.",
+                    "Or a DamageBoost SpeedKeep could be used instead of a Spike SpeedKeep with enough runspeed."
+                  ]
+                },
+                {
+                  "name": "Climb Supers In-Room SpeedKeep for Temporary Blue",
+                  "notable": true,
+                  "requires": [
+                    "canSpeedKeep",
+                    "canSlowShortCharge",
+                    {"doorUnlockedAtNode": 1},
+                    {"or": [
+                      {"and": [
+                        "canChainTemporaryBlue",
+                        {"canShineCharge": {
+                          "usedTiles": 13,
+                          "openEnd": 1
+                        }},
+                        {"spikeHits": 1}
+                      ]},
+                      {"and": [
+                        {"canShineCharge": {
+                          "usedTiles": 14,
+                          "openEnd": 1
+                        }},
+                        {"spikeHits": 2}
+                      ]},
+                      {"and": [
+                        "canChainTemporaryBlue",
+                        {"spikeHits": 4}
+                      ]}
+                    ]},
+                    "canSpeedball"
+                  ],
+                  "note": [
+                    "Using only the short runway and spike pit, use one or more SpeedKeeps to Speedball towards the Super Missile item location.",
+                    "This requires either a very short shortcharge, or a second SpeedKeep in the spikes which also resets Samus' run speed with a crouch jump before spike I-Frames expire."
```

Comment:
the 4 spike hits is only 2 speed keeps?

Reply:
yes

Reply:
Speedkeeps bounce in ball once and then are able to unmorph when they next land and take spike damage.  
Option1 is a safe runway bounce into single speedkeep
Option2 is a spike bounce into single speedkeep
Option3 is a spike bounce into speedkeep into crouch jump followed by a spike bounce into speedkeep into speedball.  And actually the first spike bounce isn't needed here.